spec/lib/hubspot-ruby_spec.rb

```diff
@@ -1,8 +1,10 @@
-describe Hubspot do
-  describe "#configure" do
-    it "delegates a call to Hubspot::Config.configure" do
-      mock(Hubspot::Config).configure({hapikey: "demo"})
-      Hubspot.configure hapikey: "demo"
+RSpec.describe Hubspot do
+  describe ".configure" do
+    it "delegates .configure to Hubspot::Config.configure" do
+      options = { hapikey: "demo" }
+
+      expect(Hubspot::Config).to receive(:configure).with(options)
```

Comment:
While we're updating this, how would you feel about moving the expectation to the bottom? i.e.

```rb
options = { hapikey: "demo" }
allow(Hubspot::Config).to receive(:configure).with(options)

Hubspot.configure(options)

expect(Hubspot::Config).to have_received(:configure)
```